src/main/java/com/github/swapbook/api/Emails.java

```diff
@@ -0,0 +1,21 @@
+package com.github.swapbook.api;
+
+import com.github.swapbook.email.EmailServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
```

Comment:
to jest tylko do testów? bo nie powinnimy udostępniać usługi wysyłającej maila z endpointu

Reply:
Jeśli to jest tylko testowy kontroler to go usuń albo dodaj tymczawowy komentarz żeby go później usunąć